challenge.py

```diff
@@ -21,8 +23,20 @@ def run():
     import unittest
 
     class ClosureSuite(unittest.TestCase):
+        def setUp(self):
+            self.results = [6, 20, 3]
```

Comment:
Esto no es un valor que vayas a pasar de un test a otro, los puedes declarar en el mismo test que propones.

Reply:
"Tests can be numerous, and their set-up can be repetitive. Luckily, we can factor out set-up code by implementing a method called setUp(), which the testing framework will automatically call for every single test we run"
https://docs.python.org/3/library/unittest.html#organizing-test-code